gulpfile.js

```diff
@@ -347,16 +268,12 @@ export async function mongoUp() {
       '-p 27017:27017',
       '-e MONGODB_ROOT_USER=root',
       '-e MONGODB_ROOT_PASSWORD=rootpw',
-      '-e MONGODB_REPLICA_SET_KEY=elmurs',
-      '-e MONGODB_REPLICA_SET_NAME=elmurs',
+      '-e MONGODB_REPLICA_SET_KEY=educandurs',
+      '-e MONGODB_REPLICA_SET_NAME=educandurs',
```

Comment:
I think these 2 should not change, or?

Reply:
Probably they do so we don't have to run on different ports all the time the 2 mong instances.

Reply:
That was the idea behind it. Otherwise we have to do a lot of cleanup when switching between elmu and educandu locally

Reply:
True, ok then.